docs/tools/android_lint.md

```diff
@@ -0,0 +1,7 @@
+# Android Lint
+[Android Lint](https://developer.android.com/studio/write/lint.html) is a code scanning tool with focus on Android. Besides code it also analyses resources
```

Comment:
I would replace `is a code scanning tool with ...` with `is a linter and static analysis tool for Android projects which can detect bugs and potential issues in code, resources and configuration files`.